### PR TITLE
New Adapter onetagBidAdapter

### PR DESCRIPTION
**Pull request by OneTagDevOps**
## Type of change

- [ ] Bugfix
- [ ] Feature
- [x] New bidder adapter   
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Does this change affect user-facing APIs or examples documented on http://prebid.org?
- [ ] Other

## Description of change

Add new Bid Adapter onetagBidAdapter

- test parameters for validating bids
```
{
  bidder: 'onetag',
  params: {
    pubId: 'your_publisher_id',         // required
    type: 'banner',                     // optional. Default 'banner'
  }
}
```

Be sure to test the integration with your adserver using the [Hello World](/integrationExamples/gpt/hello_world.html) sample page.

- contact email of the adapter’s maintainer: devops@onetag.com
- [x] official adapter submission

For any changes that affect user-facing APIs or example code documented on http://prebid.org, please provide:

- A link to a PR on the docs repo at https://github.com/prebid/prebid.github.io/

### CODE REVIEW

**Comment by bretg** [comment]
This also needs a PR against https://github.com/prebid/prebid.github.io/tree/master/dev-docs/bidders.

**Comment by OneTagDevOps** [comment]
I did a PR against https://github.com/prebid/prebid.github.io/tree/master/dev-docs/bidders, can you please check here: https://github.com/OneTagDevOps/prebid.github.io/pull/1, thanks.

**Comment by OneTagDevOps** [comment]
@jaiminpanchal27 could you please review the PR and let us know?

File: modules/onetagBidAdapter.js [hunk 1]
```diff
@@ -0,0 +1,210 @@
...
+  }
+
+  return b;
+}
+
+// Va bene così, questo file va aggiunto a prebidmaster
+export const spec = {
+
+  code: BIDDER_CODE,
+  aliases: [],
```
Comment (jaiminpanchal27) [inline modules/onetagBidAdapter.js]: remove if you don't need alias

**Comment by OneTagDevOps** [comment]
@jaiminpanchal27 Thanks for the review, I believe our last commit should have solved all the issues.

File: modules/onetagBidAdapter.js [hunk 2]
```diff
@@ -0,0 +1,210 @@
...
+}
+
+function getUserSyncs(syncOptions, serverResponses) {
+}
+
+/**
+ * Returns information about the page needed by the server in an object to be converted in JSON
+ * @returns {{location: *, referrer: (*|string), masked: *, wWidth: (*|Number), wHeight: (*|Number), sWidth, sHeight, date: string, timeOffset: number}}
+ */
+function getPageInfo() {
```
[the remark this answers is not in the document]
Comment (OneTagDevOps) [reply modules/onetagBidAdapter.js]: I believe this is not a blocking factor? We'd like to see into the util functions in the next release.

**Comment by OneTagDevOps** [comment]
@jaiminpanchal27 please let us know whether everything works fine

**Pull request merged**